### PR TITLE
do not approve: Doc/issue139 create man page for statusbar config

### PR DESCRIPTION
**Pull request by Philipp6802**
Due to our feature freeze, I left out the ```MODULES``` section. Since customising the height and width of the stautsbar is not intended, there is only the ```COLORS``` option and an example.

### CODE REVIEW

File: man/config.1 [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+When starting, Oxide has two paths it searches for config files.
+.TP
+\f[B]\[ti]/.config/oxide/config.yml\f[R]
+Home config file
+.TP
+\f[B]/etc/oxide/config.yml\f[R]
+System config file
+.SH Description
+.PP
+Define the behaviour of oxidewm.
```
Comment (gungula) [inline man/config.1:33]: To be consistent please name it Oxide wm with a capital O :)

File: man/oxide-msg.1 [hunk 2]
```diff
@@ -11,7 +25,7 @@ oxide-msg - send messages to oxidewm
 command] [\f[B]-a\f[R] argument]
 .SH DESCRIPTION
 .PP
-The \f[C]oxide-msg\f[R] is an ipc command tool allowing queriying and
+The \f[V]oxide-msg\f[R] is an ipc command tool allowing queriying and
 messaging to oxidewm via the commandline
```
Comment (gungula) [inline man/oxide-msg.1:29]: To be consistent, please name it "Oxide wm" with a capital O :)

File: man/src/oxide-bar.1.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-bar - statusbar for Oxide
+
+# DESCRIPTION
+
+Define the behavior of the statusbar for OxideWm. The config file provides the posibillity to customise the text and background color of the oxide statusbar.
```
Comment (gungula) [inline man/src/oxide-bar.1.md:11]: I guess we can name it just Oxide here, without the wm. Meanwhile this is the way we do it in the other documents :) But we should look up every document before the submission anyway.

File: man/src/oxide-bar.1.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+As same as the config file for oxide, the config for the statusbar is written in YAML.
```
Comment (gungula) [inline man/src/oxide-bar.1.md:12]: Oxide :)

File: man/src/oxide-config.1.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+**~/.config/oxide/config.yml**
+: Home config file
+
+**/etc/oxide/config.yml**
+: System config file
+
+# Description
+
+Define the behaviour of oxidewm. You can change style, layout, keybindings and more.
```
Comment (gungula) [inline man/src/oxide-config.1.md:21]: To be consistent, please name it "Oxide wm" with a capital O.

File: man/src/oxide-config.1.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+: SHIFT key
+
+## COMMANDS
+
+Commands consist of a command and optional arguments.
+
+### COMMAND
+
+Move **args** [MOVEMENT]
+: Move window
```
Comment (gungula) [inline man/src/oxide-config.1.md:52]: I know this is very picky but in the lines 52 - 92 the descriptions should be written in lowercase letters like you did in lines 93 and following to be consistent and give it a better look :)

File: man/src/oxide-config.1.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
+
+Index
+: workspace with the given index
+
+## ITERATIONS
+
+iter
+: iteraates over given number in order to change
```
Comment (gungula) [inline man/src/oxide-config.1.md:122]: typo in "iterates"

File: man/src/oxide-config.1.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+iter_cmds:
+  - iter: [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    command:
+      keys: ["M", "C", "$VAR"]
+      commands:
+        - command: GoToWorkspace
+          args: "$VAR"
+```
+
+In this example using the ALT and CONTROLL key paired with a number from one to nine, the user can go to the desired workspace.
```
Comment (gungula) [inline man/src/oxide-config.1.md:156]: typo: "CONTROL"

File: man/src/oxide-config.1.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+`$VAR` is a reference for the entered iterator.
+
+## BORDERS
+
+border_witdh
+: sets the border witdh of windows in
```
Comment (gungula) [inline man/src/oxide-config.1.md:162]: In what?

File: man/src/oxide-config.1.md [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+
+border_color
+: sets the border color and has to be entered in hexadecimal
+
+border_focus_color
+: sets the border color for focused nbdows and has to be entered in hexadecimal
```
Comment (gungula) [inline man/src/oxide-config.1.md:168]: I guess you mean "windows", not nbdows ;)

File: man/src/oxide-config.1.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+
+gap
+: gap between windows in pixels
+
+## EXECTUE
+
+exec
+: onetime execution when the window manager starts
```
Comment (gungula) [inline man/src/oxide-config.1.md:176]: typo: "one time"

File: man/src/oxide-config.1.md [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+
+exec_always
+: executes when the windosw manager starts or restarts
```
Comment (gungula) [inline man/src/oxide-config.1.md:179]: typo: "window"

File: man/oxide-msg.1 [hunk 13]
```diff
@@ -1,5 +1,19 @@
-.\" Automatically generated by Pandoc 2.9.2.1
+.\" Automatically generated by Pandoc 2.19.2
```
Comment (gungula) [inline man/oxide-msg.1:1]: Why are these changes not done in the src-file?

**Review by gungula — CHANGES_REQUESTED** [review]
Thank you for your work - i marked some small changes :) For the technical aspect i guess @FelixSchladt should give it a review, too.

File: man/oxide-msg.1 (hunk 2 above)
Comment (Philipp6802) [reply man/oxide-msg.1:29]: I don't know why this file shows up in my pull request, @FelixSchladt wrote the Oxide-msg man page.

File: man/src/oxide-config.1.md [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+: workspace with the given index
+
+## ITERATIONS
+
+iter
+: iteraates over given number in order to change
+
+# DEFAULT KEYBINDINGS
+
+TODO: DEFAULT KEYBINDINGS
```
Comment (gungula) [inline man/src/oxide-config.1.md:126]: Why is here still a TODO?

File: man/src/oxide-config.1.md [hunk 15]
```diff
@@ -0,0 +1,196 @@
...
+
+## EXAMPLES
+
+TODO: SAMPLE CONFIG
```
Comment (gungula) [inline man/src/oxide-config.1.md:130]: see line 126

**Review by gungula — CHANGES_REQUESTED** [review]

File: man/oxide-msg.1 (hunk 13 above)
Comment (Philipp6802) [reply man/oxide-msg.1:1]: maybe this happened automatically

File: man/src/oxide-bar.1.md [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-bar - statusbar for Oxide
+
+# DESCRIPTION
+
+Define the behavior of the statusbar for Oxide. The config file provides the posibillity to customize the text and background color of the oxide statusbar.
```
Comment (gungula) [inline man/src/oxide-bar.1.md:11]: typo: "possibility"

File: man/src/oxide-config.1.md [hunk 17]
```diff
@@ -0,0 +1,196 @@
+% OXIDE-CONFIG(1) oxide-config 0.1.0
```
Comment (gungula) [inline man/src/oxide-config.1.md:1]: Note: This file is definetily out of date. Please remove from this branch and only use the one on your branch doc/issue97-create-man-page-for-config-file .

**Review by gungula — CHANGES_REQUESTED** [review]

**Comment by gungula** [comment]
I suggest you resolve the requested changes and then we create a new, clean pull request together to make sure no merge-mistakes happen 👍🏼

**Comment by gungula** [comment]
new pull request (#207 )

**Pull request closed without merging**